go.mod

```diff
@@ -4,8 +4,8 @@ go 1.13
 
 require (
 	github.com/OpenPeeDeeP/depguard v1.0.1 // indirect
-	github.com/bazelbuild/bazel-gazelle v0.0.0-20181220163313-cdeedbd62467
-	github.com/bazelbuild/buildtools v0.0.0-20180226164855-80c7f0d45d7e
+	github.com/bazelbuild/bazel-gazelle v0.18.2
```

Comment:
whoo no version -> version for gazelle and klog! 😄

Reply:
This silences the build warnings about klog actually being from dims too :)

Reply:
/meow